### PR TITLE
SOMMELIER-13

### PR DESCRIPTION
**Pull request by Mixxx-a**
New tests

### CODE REVIEW

File: App/components/Switch/switch.test.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <Text>abc</Text>
+      </Switch>
+    ).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+  // it('SwitchType prop assigns correctly', () => {
+  //   const elem = themeRender(<Switch switchType='heart' />).toJSON()
+  //   fireEvent.press(elem)
+  //   expect(mockFn).toHaveBeenCalled()
+  // })
```
Comment (constantkek) [inline App/components/Switch/switch.test.tsx]: junk

Comment (Mixxx-a) [reply App/components/Switch/switch.test.tsx]: I have a problem with getting test instance of the component, so this code is a draft and will be modified when i know how to do it correctly

**Review by StanislavMyakishev — DISMISSED** [review]

File: App/components/Switch/__snapshots__/switch.test.tsx.snap [hunk 2]
```diff
@@ -36,14 +28,33 @@ exports[`Switch component Component with only necessary props matches snapshot 1
   onResponderTerminationRequest={[Function]}
   onStartShouldSetResponder={[Function]}
   style={
-    Object {
-      "opacity": 1,
-    }
+    Array [
+      Object {
+        "backgroundColor": "#ededed",
```
Comment (Maximus135) [inline App/components/Switch/__snapshots__/switch.test.tsx.snap:34]: maybe we should use color from the theme?

Comment (Mixxx-a) [reply App/components/Switch/__snapshots__/switch.test.tsx.snap:34]: IDK, it's our default switch without any passed props

**Comment by Mixxx-a** [comment]
Added testIDs in every component + Major changes in all tests. Please review

**Pull request merged**